### PR TITLE
Add Plant-it widget

### PR DESCRIPTION
**Pull request by MDeLuise**
Implements https://github.com/gethomepage/homepage/discussions/2723
Copy of https://github.com/gethomepage/homepage/pull/2890 (up-votes required number is now reached)
## Type of change
- [X] New service widget
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation only
- [ ] Other (please explain)

## Checklist:
- [X] If applicable, I have added corresponding documentation changes.
- [X] If applicable, I have reviewed the [feature](https://gethomepage.dev/latest/more/development/#new-feature-guidelines) and / or [service widget guidelines](https://gethomepage.dev/latest/more/development/#service-widget-guidelines).
- [X] I have checked that all code style checks pass using [pre-commit hooks](https://gethomepage.dev/latest/more/development/#code-formatting-with-pre-commit-hooks) and [linting checks](https://gethomepage.dev/latest/more/development/#code-linting).
- [X] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.

## Example:
Screenshot 2024-01-21 at 14 08 55

Example of usage:
```
- Plant-it:
    href: http://localhost:8085/
    description: 🪴 Self-hosted, open source gardening companion app
    widget:
      type: plantit
      url: http://localhost:8085
      key: 7QiFTqEdRguDhPNXxr83F8Obl6HpDTUTNg+7SUJdxn+s34kXfxhxHBTEA96w5NgnfS60so3XY/10YKrRXA6vow==
```

### CODE REVIEW

File: public/locales/it/common.json
Comment (shamoon) [inline public/locales/it/common.json:1]: Oh I missed this. These will get overwritten by crowdin, so changes should be made there instead =)

File: docs/widgets/services/planit.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Plant-it
+description: Plant-it Widget Configuration
+---
+
```
Comment (shamoon) [inline docs/widgets/services/planit.md:5]:
```suggestion

Learn more about [Plantit](https://github.com/MDeLuise/plant-it).

```

Our docs now include something like this.

**Review by shamoon — CHANGES_REQUESTED** [review]

File: docs/widgets/services/planit.md (hunk 1 above)
Comment (MDeLuise) [reply docs/widgets/services/planit.md:5]: Added the documentation reference, thanks.

File: public/locales/it/common.json
Comment (MDeLuise) [reply public/locales/it/common.json:1]: Got it, thanks. I'll make sure to update the translations directly in crowdin. Regarding the translations in the file, would you prefer if I remove them entirely or leave them as a reference?

File: docs/widgets/services/planit.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Learn more about [Plantit](https://github.com/MDeLuise/plant-it).
+
+API key can be created from the REST API.
+
+```yaml
+widget:
+  type: plantit
+  url: http://plant-it.host.or.ip:port
```
Comment (shamoon) [inline docs/widgets/services/planit.md:13]:
```suggestion
  url: http://plant-it.host.or.ip:port # api port
```

**Pull request merged**